kobo/settings/base.py

```diff
@@ -475,6 +475,10 @@
         'Enable automatic deletion of attachments for users who have exceeded '
         'their storage limits.'
     ),
+    'ANON_EXPORTS_CLEANUP_AGE': (
```

Comment:
Let's be consistent with other similar variables. e.g.: `*_GRACE_PERIOD`. 
Also I would write `ANONYMOUS_EXPORTS_...` .